network/benches/network_bench.rs

```diff
@@ -310,7 +310,7 @@ fn rpc_bench(b: &mut Bencher, msg_len: &usize) {
                 req.clone(),
             ));
         }
-        while let Some(res) = block_on(requests.next()) {
+        while let Some(res) = runtime.block_on(requests.next()) {
```

Comment:
we need to ensure there are enough slots in the queue (`NUM_MSGS <= capacity`) since `libra_channel` will drop messages under congestion for both rpc and direct-send, although I think the rpc bench will at least panic if one of the requests is dropped.